src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/Internal/HttpRequestReader.cs

```diff
@@ -111,24 +141,98 @@ public async Task ReadRequestAsync(LogRecord logRecord, HttpRequestMessage reque
             logRecord.RequestBody = await _httpRequestBodyReader.ReadAsync(request, cancellationToken)
                 .ConfigureAwait(false);
         }
+
+        logRecord.QueryParameters = GetQueryParameters(request);
+
+        var sb = new StringBuilder();
+        foreach (var kvp in logRecord.QueryParameters)
+        {
+            if (sb.Length > 0)
+            {
+                _ = sb.Append('&');
+            }
+
+            _ = sb.Append(kvp.Key).Append('=').Append(kvp.Value);
+        }
+
+        var queryString = sb.ToString();
+
+        logRecord.FullUri = string.IsNullOrEmpty(queryString)
+            ? null
```

Comment:
```suggestion
            ? $"{logRecord.Host}/{logRecord.Path}"
```

Shouldn't we include here a request path without a query string when the latter is null?

Reply:
We can, don't see much of issue here. Although it's just a combination of 2 existing metrics,  so I'm not sure that it's really useful

Reply:
I'll emit it, since there was another suggestion for it

Reply:
Done